src/engine/DisjunctionConstraint.cpp

```diff
@@ -19,9 +19,24 @@
 #include "Statistics.h"
 
 DisjunctionConstraint::DisjunctionConstraint( const List<PiecewiseLinearCaseSplit> &disjuncts )
-    : _disjuncts( disjuncts )
-    , _feasibleDisjuncts( disjuncts )
+    : ContextDependentPiecewiseLinearConstraint( disjuncts.size() )
+    , _disjuncts( disjuncts.begin(), disjuncts.end() )
+    , _feasibleDisjuncts( disjuncts.size(), 0 )
```

Comment:
why not just _feasibleDisjuncts( disjuncts )? Instead of the loop in the body of the function

Reply:
The data structure differ now, it used to be a List and now its a Vector to utilize indexing using PhaseStatus. I didn't want to meddle with outer interface for now, but if you want I can go fix that where needed.